setup.py

```diff
@@ -60,7 +60,7 @@
     author="Google LLC",
     author_email="googleapis-packages@google.com",
     license="Apache 2.0",
-    url="https://github.com/GoogleCloudPlatform/google-cloud-python",
+    url="https://github.com/GoogleCloudPlatform/python-webrisk",
```

Comment:
should be googleapis not googlecloudplatform

Reply:
🤦‍♀ Thank you!